mosquitto/docker-compose.yml

```diff
@@ -2,12 +2,11 @@
 version: "3"
 services:
   mosquitto:
-    image: hub.foundries.io/<your factory>/mosquitto:latest
-    tmpfs:
-      - /run
-      - /var/lock
-      - /var/log
+    image: eclipse-mosquitto:1.6.12
+    volumes:
+      - /var/run/secrets/:/mosquitto/secrets
+      - ./run.sh:/run.sh:ro
+    command: [/bin/sh, /run.sh]
```

Comment:
So you can make this: `command: [/run.sh]`. I bet the reason it didn't work is because you didnt' "chmod +x ./run.sh" before you checked it into git. You may also need the shebang in run.sh to be "/bin/sh" I don't know if bash is in that container or not.

Reply:
Just did the change that you asked for on my own factory:

docker-compose.yml
`    command: [/run.sh]     `

File inside the container
`/ # cat run.sh 
#!/bin/sh
`

Ps inside the container
`PID   USER     TIME  COMMAND
    1 root      0:00 {run.sh} /bin/sh /run.sh
   12 mosquitt  0:00 /usr/sbin/mosquitto -c /mosquitto/secrets/mosquitto.conf
   21 root      0:00 sleep 5
   22 root      0:00 sh
   28 root      0:00 ps
`

Will update the PR.